libs/common/src/models/export/fido2key.export.ts

```diff
@@ -30,6 +32,8 @@ export class Fido2KeyExport {
     view.counter = parseInt(req.counter);
     view.rpName = req.rpName;
     view.userDisplayName = req.userDisplayName;
+    view.discoverable = !!req.discoverable;
```

Comment:
@djsmith85 asked me about the `!!` operation, and I think we should change this to be consistent with how we convert Domain -> View
```suggestion
    view.discoverable = discoverable === "true"
```